src/treelearner/leaf_splits.hpp

```diff
@@ -60,47 +60,64 @@ class LeafSplits {
   }
 
   /*!
-  * \brief Init splits on current leaf, it will traverse all data to sum up the results
-  * \param gradients
-  * \param hessians
-  */
+   * \brief Init splits on current leaf, it will traverse all data to sum up the
+   * results \param gradients \param hessians
+   */
```

Comment:
```suggestion
   * \brief Init splits on the current leaf, it will traverse all data to sum up the results
   * \param gradients
   * \param hessians
   */
```